java/server/src/org/openqa/selenium/grid/distributor/Distributor.java

```diff
@@ -130,23 +130,28 @@ protected Distributor(
 
     Json json = new Json();
     routes = Route.combine(
-      post("/session").to(() -> req -> {
-        CreateSessionResponse sessionResponse = newSession(req);
-        return new HttpResponse().setContent(bytes(sessionResponse.getDownstreamEncodedResponse()));
-      }),
-      post("/se/grid/distributor/session")
-          .to(() -> new CreateSession(this)),
-      post("/se/grid/distributor/node")
-          .to(() -> new AddNode(tracer, this, json, httpClientFactory)),
-      delete("/se/grid/distributor/node/{nodeId}")
-          .to(params -> new RemoveNode(this, new NodeId(UUID.fromString(params.get("nodeId"))))),
-      get("/se/grid/distributor/status")
-          .to(() -> new GetDistributorStatus(this))
-          .with(new SpanDecorator(tracer, req -> "distributor.status")));
+        post("/session").to(() -> req -> {
+          CreateSessionResponse sessionResponse = newSession(req);
+          return new HttpResponse()
+              .setContent(bytes(sessionResponse.getDownstreamEncodedResponse()));
+        }),
+        post("/se/grid/distributor/session")
+            .to(() -> new CreateSession(this)),
+        post("/se/grid/distributor/node")
+            .to(() -> new AddNode(tracer, this, json, httpClientFactory)),
+        post("/node/{nodeId}/drain")
```

Comment:
Prefix the URL with `/se/` to stay in conformance with the webdriver spec, please.